index.html

```diff
@@ -543,6 +543,99 @@ <h6>JWT Decoding</h6>
       </section>
     </section>
 
+    <section id="production">
+      <h2>Production</h2>
+
+      <p>
+        This section describes how to produce a VC-JWT encoded
+        <code>VerifiableCredential</code> from a <code>Credential</code>.
+      </p>
+
+      <p class="advisment">
+        There are currently 2 competing solutions to this problem. It is a goal
+        of the v2 work to define a breaking change to the specification that
+        reduces production rules to a single, simple, set of instructions that
+        any implementer can easily meet if they posses a software library
+        supporting [[RFC7515]] or [[RFC7519]].
+      </p>
+
+      <section>
+        <h3>In addition to...</h3>
+
+        <p>
+          There are several members (claims) of the
+          <code>Credential</code> which will need to be translated to their JOSE
+          form, and included next to the <code>vc</code> or
+          <code>vp</code> member in the <code>payload</code>.
```

Comment:
Can we add an explanation why do these members need to be translated to the equivalents of the JWT claims?
iss, nbf, all of them are optional in RFC 7519 itself. They only become required when the JWT is used for a specific purpose, and the only place I can think of right now is an ID Token.

Reply:
hmm, yes, we should do that... but I am not sure we need to do that now... My goal was to write the production rules for all interpretations of v1 first... Can I convert this comment to an issue for discussion?

Reply:
yes.

Reply:
https://github.com/w3c/vc-jwt/issues/15#issue-1374577341